packages/workflows/src/utils/composer/type.ts

```diff
@@ -54,9 +65,21 @@ export type CreateWorkflowComposerContext = {
   ) => TOutput
 }
 
+/**
+ * The step's context.
+ */
 export interface StepExecutionContext {
+  /**
+   * The container used to access resources, such as services, in the step.
+   */
   container: MedusaContainer
```

Comment:
first iteration we keep it like it and force people behviour of providing this, but later we can tell them that in fact you can pass anything you want